main.go

```diff
@@ -93,6 +94,12 @@ func main() {
 			Usage:  "Slack webhook URL",
 			EnvVar: "LAIKA_SLACK_WEBHOOK_URL",
 		},
+		cli.StringFlag{
+			Name: "route-prefix",
+			Usage: "Route prefix",
+			EnvVar: "LAIKA_ROUTE_PREFIX",
+			Value: "/laika",
```

Comment:
The default value should be _the empty string_, because we don't always need to prefix the routes